### PR TITLE
Add Learn page

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Migrate `/learn` page

## Related Issue
https://www.notion.so/efdn/Migrate-React-pages-8acee0dc80d04e55b109e9852c20ef2f?pvs=4

## Preview links
https://deploy-preview-175--ethereum-org-fork.netlify.app/en/learn

### CODE REVIEW

File: src/pages/learn.tsx [hunk 1]
```diff
@@ -0,0 +1,722 @@
...
+  const requiredNamespaces = getRequiredNamespacesForPage("/learn")
+  const lastDeployDate = getLastDeployDate()
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+    },
+  }
+}) satisfies GetStaticProps<SSRConfig>
```
Comment (pettinarip) [inline src/pages/learn.tsx]: `lastDeployDate` missing in prop types

File: src/pages/learn.tsx [hunk 2]
```diff
@@ -0,0 +1,722 @@
...
+          eventCategory: "learn hub hero buttons",
+          eventAction: "click",
+          eventName: "lets get started",
+        },
+        variant: "solid",
+      },
+    ],
+  }
+
+  const height200 = { h: 200, style: { width: "auto" } }
```
Comment (pettinarip) [inline src/pages/learn.tsx]: Next Image only supports `height`.
```suggestion
  const height200 = { height: 200, style: { width: "auto" } }
```

Comment (pettinarip) [reply src/pages/learn.tsx]: You can see here the impact of this change:

current version (will try to fetch the source image)
image

using `height`
image

**Review by pettinarip — APPROVED** [review]
Nice @wackerow thanks!

Pushed a commit to fix the mentioned comments. LMK what do you think.

File: src/pages/learn.tsx (hunk 2 above)
Comment (wackerow) [reply src/pages/learn.tsx]: Okay thanks... same mistake on /staking page... didn't realize `w` was different than `width` here... will patch

**Comment by wackerow** [comment]
Thanks for the patch @pettinarip, LGTM!

**Pull request merged**